Headers/CoreGraphics/CGContext.h

```diff
@@ -562,6 +578,19 @@ CGRect CGContextConvertRectToDeviceSpace(CGContextRef ctx, CGRect rect);
 
 CGRect CGContextConvertRectToUserSpace(CGContextRef ctx, CGRect rect);
 
+bool CGContextPathContainsPoint(CGContextRef c,
+    CGPoint point, CGPathDrawingMode mode);
+
+/* CoreGraphics Private APIs */
```

Comment:
This function gets exported by the Apple implementation as well. Having it here is fine.